bifrost/app/blog/blogs/best-arize-alternatives/src.mdx

```diff
@@ -100,31 +107,61 @@ Arize Phoenix is an open-source LLM observability tool that focuses on providing
 
 Arize Phoenix excels in its evaluation capabilities and is well-suited for data scientists and ML engineers working on complex LLM projects. However, it lacks some of the developer-friendly features that Helicone offers, such as self-hosting options, user tracking, and user feedback collection. Arize Phoenix's pricing model may also be less flexible compared to Helicone's tiered approach.
 
+## Why Companies Choose Arize Phoenix Over Helicone? 
+
+- **Robust Evaluation Capabilities**: Ideal for data scientists focused on model performance.
+- **Integration with ML Workflows**: Seamlessly fits into existing machine learning pipelines.
+
 ---
 
-### So, which LLM observability tool suits you better?
+## So, which LLM observability tool suits you better?
+
+Both Helicone and Arize Phoenix offer powerful features for LLM observability, but they cater to slightly different audiences.
+
+### Choose Helicone if you:
+* Require self-hosting options for data control and compliance.
```

Comment:
No, we also have SOC2, the ability to omit storing bodies

Reply:
what do you suggest here?

Reply:
If they're concerned about control and compliance, Helicone has SOC2, the ability to omit storing bodies and self-hosting options